pkg/k8s/from_yaml.go

```diff
@@ -14,11 +14,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// instead of pkg/maya/maya.go
+// Package k8s defines all the k8s related codebase.
```

Comment:
@princerachit 
Please give some insights here.
I feel this comment can be improved.

Reply:
Okay @sonasingh46 👍

Reply:
Hi @sonasingh46, I've updated the partial improvement of the comment, hopefully @princerachit can give some insights/feedback :) thank you so much!